server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -81,17 +94,25 @@ async def fetch_releases():
     data = await asyncio.gather(
         fetch_prs(), fetch_comments(), fetch_commit_authors(), fetch_commit_committers(),
         fetch_reviews(), fetch_releases())
-    users = {}
+
+    stats = defaultdict(dict)
     for rows, key in zip(data, ("prs", "commenter", "commit_author", "commit_committer",
                                 "reviewer", "releaser")):
         for row in rows:
-            users.setdefault(row[0], {})[key] = row[1]
-    try:
-        del users[None]
-    except KeyError:
-        pass
```

Comment:
This is a real case in production that is unfortunately not covered in the tests. We should return it back.

Reply:
ah yeah sorry, I should have definitely seen in the changes, I replaced it `pop` and I forgot it after further rewrite 🤦.